pkgs/development/python-modules/sasmodels/default.nix

```diff
@@ -0,0 +1,23 @@
+{lib, fetchgit, buildPythonPackage, pytest, numpy, scipy, matplotlib, docutils}:
+
+buildPythonPackage rec {
+  name = "sasmodels-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.